DMTN-148.tex

```diff
@@ -50,9 +50,255 @@
 % ADD CONTENT HERE
 % You can also use the \input command to include several content files.
 
+\section{Intro}
+
+The goal of this tech note is to provide a plan for the handling of
+calibration products in the Gen-3 Butler environment.  There are a
+large number of these products currently in use, but many of them do
+not have a clearly defined process to define, ingest, test, and
+deprecate them.  The lack of documentation on how these processes work
+has led to many implementations of similar paths, further complicating
+the issue.
+
+
+\section{The End State}
+
+The final state that we wish to get to has the following properties:
```

Comment:
What I'd _really_ like to see is a little more justification of _why_ this is the desired end-state. I mean, it sounds plausible enough, but are we sure we're covering all the bases? In particular, do we have a common mental model of how this information will be used and managed _by the operations team_, which should really be the driving consideration here?

I'm a bit torn here because I'm reluctant to send @czwa off on a substantial requirements gathering exercise, but I can't help but feel that we ought to have some sort of driving use-cases, rather than just the statement of the solution without any real motivation.

Reply:
We did get initial feedback from the calibration scientist but it would be nice if we got feedback from the calibration products team.